core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -352,8 +353,11 @@ object ConsumerGroupCommand {
       if (consumer != null) consumer.close()
     }
 
-    private def createAdminClient(): AdminClient =
-      AdminClient.createSimplePlaintext(opts.options.valueOf(opts.bootstrapServerOpt))
+    private def createAdminClient(): AdminClient = {
+      val props = if (opts.options.has(opts.commandConfigOpt)) Utils.loadProps(opts.options.valueOf(opts.commandConfigOpt)) else new Properties()
+      props.put(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG, opts.options.valueOf(opts.bootstrapServerOpt))
+      AdminClient.create(scala.collection.JavaConversions.propertiesAsScalaMap(props).map(x => x._1 -> x._2).toMap)
```

Comment:
And even better would be to add a separate `create` in AdminClient which accepts Properties.